values.yaml

```
@@ -1151,6 +1151,7 @@ nginx:
   http_listen_port: 80
   config:
     dnsResolver: kube-dns.kube-system.svc.cluster.local
+    dnsTtl: 10s
```

This is a nitpick since I acknowledge we already have [multiple](https://github.com/cortexproject/cortex-helm-chart/blob/c7a7f4e198c19a9679504ea062c61795fba765ef/values.yaml#L382) [styles](https://github.com/cortexproject/cortex-helm-chart/blob/c7a7f4e198c19a9679504ea062c61795fba765ef/values.yaml#L298) in this project, but I prefer the style of capitalizing acronyms.
```suggestion
    dnsTTL: 10s
```